webapp/channels/src/components/advanced_create_comment/advanced_create_comment.tsx

```diff
@@ -838,6 +843,16 @@ class AdvancedCreateComment extends React.PureComponent<Props, State> {
         const ctrlAltCombo = Keyboard.cmdOrCtrlPressed(e, true) && e.altKey;
         const shiftAltCombo = !Keyboard.cmdOrCtrlPressed(e) && e.shiftKey && e.altKey;
 
+        // fix for FF not capturing the paste without formatting event when using ctrl|cmd + shift + v
+        if (e.key === KeyCodes.V[0] && ctrlOrMetaKeyPressed) {
+            if (e.shiftKey) {
+                this.isNonFormattedPaste = true;
+                this.timeoutId = window.setTimeout(() => {
```

Comment:
Wonder why we use `window.setTimeout` instead of directly `setTimeout`.

Reply:
to make TS happy ;)

Reply:
One way is to type the timeout as a `NodeJS.Timeout`. But this is also fine (unless @hmhealey knows any reason why it would not be).